src/theme/components/hero.ts

```diff
@@ -105,8 +105,65 @@ const getSecondaryVariantStyles = (bgColor: string = "") => {
     },
   };
 };
+const getTextColor = (type, mode, foregroundColor, isDarkText) => {
+  const prefix = mode === "dark" ? "dark.ui." : "ui.";
+  const colorLight = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.${type}`
+    : `${prefix}typography.inverse.${type}`;
+  const colorDark = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `${prefix}typography.inverse.${type}`
+    : `${prefix}typography.${type}`;
+  const colorLink = foregroundColor
+    ? foregroundColor
+    : isDarkText
+    ? `ui.link.primary`
+    : `dark.ui.link.primary`;
+  const finalColor =
+    type === "link" ? colorLink : mode === "dark" ? colorDark : colorLight;
+  return finalColor;
+};
+const getLinkColor = (state, mode, foregroundColor, isDarkText) => {
+  let finalColor;
+  const isDarkMode = mode === "dark" ? true : false;
+  switch (state) {
+    default:
+      finalColor = foregroundColor
+        ? foregroundColor
+        : isDarkText
+        ? `var(--nypl-colors-ui-link-primary) !important`
+        : isDarkMode
+        ? `dark.ui.link.primary`
+        : `dark.ui.link.primary`;
```

Comment:
Both of these are the same but I don't think they should be. Wouldn't it be checked against `isDarkMode` here so whether it's true or false you'll get the same value.

Reply:
Yes, this is a little confusing to see.  However, the default background color (without settign the `backgroundColor` prop) is dark for both the light mode and dark mode variants of the Hero component, so the default color of the links should always be that one value (lighter).  

That being the case, you might ask, why even use a ternary condition her?  I left that in there in case we did eventually need to differentiate the color values for light mode and dark mode.  

But in the end, I know you left another comment about nesting too many ternary conditions.  Based on that, I have removed this last ternary condition.  I also added a comment to somewhat explain the reasoning.